Conch/lib/Conch/Validation/SwitchPeers.pm

```diff
@@ -26,14 +26,21 @@ sub validate {
 
 	my $rack_slots = $device_location->datacenter_rack->slots;
 
-	my @peer_ports =
-		$self->_calculate_switch_peer_ports( $device_location->rack_unit,
-		$rack_slots );
-
 	my @eth_nics =
 		map { $data->{interfaces}->{$_} }
 		grep { $_ =~ /eth/ } ( keys $data->{interfaces}->%* );
 
+	# XXX This is awful. I'm sorry.
```

Comment:
Let's find a way to do this without another "this is awful" comment in the codebase. I'll take a look.

Reply:
what is awful about this?